blackbox/docs/README.rst

```diff
@@ -0,0 +1,71 @@
+|                        | information as well as a list of changes.          |
++------------------------+----------------------------------------------------+
+| Installation           | THIS SECTION IS CHANGING SOON                      |
++------------------------+----------------------------------------------------+
+| Configuration          | Documentation for configuration settings, split    |
+|                        | into sections by theme.                            |
++------------------------+----------------------------------------------------+
+| Command Line Interface | Documentation for the ``crate`` command.           |
++------------------------+----------------------------------------------------+
+| CrateDB SQL            | THIS SECTION IS CHANGING SOON                      |
++------------------------+----------------------------------------------------+
+| Blob Support           | THIS SECTION IS CHANGING SOON                      |
++------------------------+----------------------------------------------------+
+| Plugins                | Primary documentation for plugins, split into      |
+|                        | documents by plugin.                               |
++------------------------+----------------------------------------------------+
+| Client/Server Protocol | Documentation for client/server protocols, split   |
+|                        | into documents by protocol.                        |
++------------------------+----------------------------------------------------+
+| Enterprise Features    | A list of each enterprise feature, with a link     |
+|                        | to the primary documentation.                      |
++------------------------+----------------------------------------------------+
+| Administration         | Primary documentation for any feature primarily of |
+|                        | use to a database administrator, split into        |
+|                        | sections by theme.                                 |
++------------------------+----------------------------------------------------+
+| JMX Monitoring Plugin  | THIS SECTION IS CHANGING SOON                      |
++------------------------+----------------------------------------------------+
+| Appendix               | Supplementary information.                         |
++------------------------+----------------------------------------------------+
+
+How to Document a Feature
+=========================
+
+A single feature may have three types of documentation:
+
+- Primary documentation
+- Configuration documentation
+- SQL syntax documentation
+
+Primary documentation is located under one of the following sections:
+
+- *CrateDB SQL*
+- *Administration*
+- *Plugins*
+
+Primary documentation should not include a full configuration settings
+reference or a full SQL syntax reference. Instead, full references should be
+added to *Configuration* or *SQL Syntax* sections.
```

Comment:
Can you specify what the `Primary documentation` should cover.
In my opinion:
* Understanding the feature.
* Giving basic information on how to use the feature, so that the user is able to use it.

Reply:
@joemoe pushed a fixup